Nuget/Ploeh.AutoFixture.Xunit2.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+    <metadata>
+        <id>AutoFixture.Xunit2</id>
+        <version>2.2</version>
+        <title>AutoFixture with xUnit.net v2 data theories</title>
+        <authors>Mark Seemann</authors>
```

Comment:
@TeaDrivenDev, you are welcome to replace my name with yours here, if you'd like!